packages/file/lib/src/backends/record_replay/recording_proxy_mixin.dart

```diff
@@ -90,6 +90,10 @@ abstract class RecordingProxyMixin implements ProxyObject, ReplayAware {
   @protected
   Stopwatch get stopwatch;
 
+  // This check is used in noSuchMethod to detect if this code is running in a
+  // Dart 1 runtime, or Dart 2.
+  bool get _runningDart1Runtime => <dynamic>[] is List<String>;
```

Comment:
Can you add a TODO referencing dart-lang/sdk#33459 to remove this once the underlying issue is fixed?

Reply:
This one actually doesn't have to do with https://github.com/dart-lang/sdk/issues/33459. But I can add a TODO that we can remove once the minimum Dart SDK version is... 2.0 or something.

Reply:
Done.